evap/staff/tests/test_importers.py

```diff
@@ -775,8 +818,38 @@ def test_existing_participation(self):
         )
         self.assertFalse(importer_log.has_errors())
 
+    @override_settings(IMPORTER_COURSE_NAME_SIMILARITY_WARNING_THRESHOLD=0.8)
+    def test_course_name_with_typo(self):
+        # Add a typo in one english course name as well
+        input_data = deepcopy(excel_data.test_enrollment_data_filedata)
+        self.assertEqual(input_data["MA Belegungen"][1][7], "Build")
+        input_data["MA Belegungen"][1][7] = "Biuld"
+        input_data["MA Belegungen"][1][6] = "BauenWithTypo"
+
+        excel_content = excel_data.create_memory_excel_file(input_data)
+
+        args = (excel_content, self.semester, self.vote_start_datetime, self.vote_end_date)
+        importer_log_test = import_enrollments(*args, test_run=True)
+        importer_log_notest = import_enrollments(*args, test_run=False)
+
+        self.assertEqual(importer_log_test.warnings_by_category(), importer_log_notest.warnings_by_category())
+        self.assertListEqual(
```

Comment:
```suggestion
        self.assertEqual(
```

... and this doesn't use `assertErrorsAre` because there could be more that we don't care about, correct?

Reply:
Hmm, yeah, I thought about adding `assertWarningAre`, but I think with warnings the use case is a bit different. With errors, we often want to assert that no other errors happens, especially that no "Import aborted after uncaught exception" error is generated.
With warnings, I'd say we most often don't really care if other warnings trigger. I wouldn't want this test to break if we add a "suspiciously low enrollment count" warning or whatever.

Thoughts / fine?